lib/services.identity.js

```diff
@@ -516,11 +516,6 @@ api._createIdentity = function(options, req, callback) {
     }
     // result details
     var details = {identity: results.createIdentity};
-    // schedule identity created event
-    bedrock.events.emitLater({
-      type: 'bedrock.Identity.created',
```

Comment:
If we do what dave actually said to do (listen for postInsert and emit identity.created from it) then this isn't an issue, but still going to want to look at updating bedrock-mail.